src/components/InlineSystemMessage.js

```diff
@@ -9,7 +9,11 @@ import Icon from './Icon';
 
 const propTypes = {
     /** Error to display */
-    message: PropTypes.string.isRequired,
+    message: PropTypes.string,
+};
+
+const defaultProps = {
+    message: '',
```

Comment:
I think yes? We can design the component however we want. In this case, if an undefined `message` is passed here we will default to empty string and then return `null` because `props.message.length === 0`.